bin/analysis.cpp

```diff
@@ -145,6 +147,10 @@ class chainNames
 
 	bool checkValidMode(std::string mode)
 	{
+	        if(mode.find("WRto")!=std::string::npos) {
```

Comment:
std::string::npos can be replased by _ENDSTRING defined at the beginning of the file